vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -122,7 +122,10 @@ public RichTextEditor() {
      *            the initial value in Delta format, not {@code null}
      *
      * @see #setValue(Object)
+     *
+     * @deprecated Since 23.3, this API is deprecated.
```

Comment:
```suggestion
     * @deprecated Since 23.3, this API is deprecated. Use {@link #setValue(String)} instead.
```
Could add a note what should be used instead. Obvious in this case, but a good practice in general if there is an alternative to what is being deprecated.